google_assistant/README.md

```diff
@@ -69,6 +69,7 @@ In case you've found a bug, please [open an issue on our GitHub][issue].
 [armv7-shield]: https://img.shields.io/badge/armv7-yes-green.svg
 [dialogflow-integration]: https://www.home-assistant.io/integrations/dialogflow/
 [discord]: https://discord.gg/c5DvZ4e
+[documentation]: https://github.com/home-assistant/hassio-addons/blob/master/google_assistant/DOCS.md
```

Comment:
Is it not better to reference a relative link since the file is in the same repo?

With a full link to GitHub, I think that the user will be redirected directly into GitHub instead of staying on the add-on page in the Supervisor.
I am not sure if this is the expected behaviour. 🤔

Reply:
With a relative link, you get the issue you had with the dual sidebars ;)

Reply:
True

Reply:
But I'm still unsure what is supposed to be the "UI support" with the `DOCS.md` file :P

Reply:
There will be a new tab on the addon:
![image](https://user-images.githubusercontent.com/15093472/81505305-77eb4300-92ee-11ea-8dac-b111630f472d.png)

Reply:
Oh great!!

Do you think it's possible to make it that when you click on the `Click here for the full documentation`, it redirects the UI to the documentation tab?

If not, a direct link to GitHub is great.

Reply:
No that is not possible, that would break the link if browsed on Github.